src/Badoo/SoftMocks.php

```diff
@@ -819,6 +819,14 @@ public static function ignoreFunction($function)
         SoftMocksTraverser::ignoreFunction(ltrim($function, '\\'));
     }
 
+    /**
+     * @param string $function - Allow to mock $function
+     */
+    public static function allowFunction($function)
```

Comment:
You can already override internal functions if you really need to by calling SoftMocks::setRewriteInternal(true).

Reply:
The idea was we need only couple function to redefine, but the library redefines many of them.
For us it's easier to say which functions to allow.

Reply:
Yeah but you can (should) define your own wrappers on top of SoftMocks already and you can apply filters on that level. I think that there should be better reasons to add new public methods than this.